theme/theme.go

```diff
@@ -61,6 +61,39 @@ func DarkTheme() fyne.Theme {
 	return theme
 }
 
+// shade will darken a light color and lighten a light color by the given % value
```

Comment:
% or pct should explain it's expected range better.
In the code it clamps it to < 50, but I couldn't say why

Reply:
It comes from the material design recomendations. The actual percents to lighten is twice the percent to darken. The shade for focus is 12% darken, but 24% lighten. In this way we need only one constant and the doubling is done in Shade(). I have improved the comments.